### ISSUE TITLE
Make a "Case Cancelled Check" script

### ISSUE THREAD

**Issue by RasmusBurge-CG** [Clinical-Genomics/cg_lims#310]
Lims Issue
---------------

For definitions of headings below see [Basic Concepts](https://atlas.scilifelab.se/infrastructure/lims/basic_concepts/).

**Work Flow:** Twist, WGS
**Protocol:** RC, pooling and maybe aliquotation steps 
**Step:** Aggregation (RC), Pooling and Hyb, qPCR. 

**Description:** 

Samples are often connected to each other and the final analysis is dependent on that all samples in one case make it to the final step without cancellation. Therefore it is important to know if any sample related to one sample is canceled, because it might lead to the cancellation of the entire case and all samples in it. This is easy to miss and samples of a canceled case have been prepped and analysed to no purpose. We suggested that we make a script capable of taking the samples active in a LIMS step and see if there are any cancelled samples related to that sample and if that is the case, give a warning. 

Mock of the code:
```
for case in step:
    for samples in case:
           if lower( sample.udf["cancelled"] ) ==  "yes":
                    prompt.warning
                    write to log file
                    continue()
```
Mock of test:

* test case without cancelled sample
* test case with cancelled sample

Additional thoughts
- Might be good to pick up on weird values in the cancelled udf, like yees or similar.

**Comment by RasmusBurge-CG** [comment]
So, this was not as easy as first expected, as many things are. We will have to find a way to speak to statusDB in order to get the case information. Otherwise, we can only give a warning if a sample in the project has been canceled, might not be soo helpful..

**Comment by AnnaZetterlund** [comment]
Ok, cancelled samples are quite common and may not be relevant in many cases. Could be worth talking to someone in Prod Bioinfo if they have any ideas?

**Comment by RasmusBurge-CG** [comment]
Yes, me and Karl Svärd have talked about it and he will look see if we have done something similar before. :)
